packages/web/src/scss/components/Modal/_theme.scss

```diff
@@ -18,10 +18,10 @@ $common-padding-x-tablet: tokens.$space-800;
 $dialog-width: 100%;
 $dialog-width-tablet: 640px;
 $dialog-width-desktop: 680px;
-$dialog-height: min-content;
-$dialog-height-tablet: min-content;
+$dialog-height: var(--modal-preferred-height-mobile, min-content);
+$dialog-height-tablet: var(--modal-preferred-height-tablet, min-content);
 $dialog-max-height: calc(100dvh - #{tokens.$space-1100});
```

Comment:
Maybe make mobile max height customizable too? But does it make sense? 🤔

Reply:
I don't know. We can add it later when we find a use case for it.

Reply:
OK.